src/api/export.ts

```diff
@@ -140,11 +144,13 @@ async function execute(emitter: EventEmitter, {
 
 					if (href) {
 						const url = new URL(href, base.href);
-						if (url.origin === origin) urls.push(url);
```

Comment:
Hi, I think the `urls` declaration is not needed anymore in line 127

Reply:
good catch, thanks — removed